src/content/roadmap/index.md

```diff
@@ -0,0 +1,108 @@
+
+## When will the roadmap be finished?
+
+The current roadmap contains some upgrades that will be implemented in the next 6 months (e.g. staking withdrawals) and others that will probably take 5-10 years to be fully implemented (e.g. quantum resistance). The precise timing of each upgrade is very difficult to predict as many roadmap items are worked on in parallel and develop at different speeds. The relative urgency of different upgrades can also change over time depending on external factors (e.g. a sudden leap in the performance and availability of quantum computers may make quantum resistant cryptography more urgent).
+
+One way to think about Ethereum development is by analogy to biological evolution. A network that is able to adapt to new challenges and maintain fitness is more likely to succeed that one that is resistant to change, although as the network becomes more and more performant, scalable and secure fewer changes to the protocol will be required.
+
+## Do I have to do anything when there is an upgrade?
+
+Upgrades tend not to impact end-users except by providing better user-experiences and a more secure protocol and perhaps more <i>options</i> for how to interact with Ethereum. End uses are not required to actively participate in an upgrade, nor are they required to do anything to secure their assets. Node operators will need to update their clients to prepare for an upgrade. Some upgrades may lead to changes for application developers. For example, history expiry upgrades may lead application developers to grab historical data from new sources.
+
+## What about the verge, the splurge etc?
+
+Vitalik proposed a vision for the Ethereum roadmap that was organized into several categories linked by their effects on Ethereum's artchitecture. It included:
+
+- The Merge: upgrades relating to the switch from proof-of-work to proof-of-stake
+- The Surge: upgrades related to scalability by rollups and data sharding
+- The Scourge: upgrades related to censorship resistance, decentralization and protocol risks from MEV
+- The Verge: upgrades related to verifying blocks more easily
+- The Purge: upgrades related to reducing the computational costs of running nodes and simplifying the protocol
+- The Splurge: other upgrades that don't fit well into the previous categories.
+
+We decided not to persist this terminology on this page because we wanted to use a simpler and more user-centric model.
```

Comment:
```suggestion
We decided not to use this terminology because we wanted to use a simpler and more user-centric model. Although we use user-centric language, the vision remains the same as the one proposed by Vitalik.
```